src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -774,6 +813,41 @@ public function generateStepsPhp($actionObjects, $generationScope = TestGenerato
             }
 
             switch ($actionObject->getType()) {
+                case "helper":
+                    if (!in_array($customActionAttributes['class'], $this->customHelpers)) {
+                        $this->customHelpers['$' . $stepKey] = $customActionAttributes['class'];
+                    }
+
+                    $arguments = [];
+                    $classReader = new \Magento\FunctionalTestingFramework\Helper\Code\ClassReader();
+                    $parameters = $classReader->getParameters(
+                        $customActionAttributes['class'],
+                        $customActionAttributes['method']
+                    );
+                    $typesToQuote = ['string', 'float'];
+                    foreach ($parameters as $parameter) {
+                        if (isset($customActionAttributes[$parameter['variableName']])) {
+                            $value = $customActionAttributes[$parameter['variableName']];
+                            $arguments[] = $this->addUniquenessFunctionCall(
+                                $value,
+                                in_array($parameter['type'], $typesToQuote) || $parameter['type'] === null
+                            );
+                        } elseif (!$parameter['isOptional']) {
+                            throw new TestFrameworkException(
+                                'Argument \'' . $parameter['variableName'] . '\' for method '
+                                . $customActionAttributes['class'] . '::' . $customActionAttributes['method']
+                                . ' is not found.'
```

Comment:
Method throws an exception when the first argument in the signature is missed. Would it be helpful to gather all missed arguments and throw an exception as in case of actionGroup like this - 
` Arguments missed (adminUserName, adminPassword) for actionGroup "LoginAsAdmin" in Test "AdminLoginTest"`

Reply:
Done